docs/csharp/asynchronous-programming/start-multiple-async-tasks-and-process-them-as-they-complete.md

```diff
@@ -23,7 +23,7 @@ You can follow this tutorial by using one of the following options:
 
 ## Create example application
 
-Create a new .NET Core console application that targets .NET 6.0. You can create one by using the [dotnet new console](../../../../core/tools/dotnet-new-sdk-templates.md#console) command or from Visual Studio.
+Create a new .NET Core console application that targets .NET 6.0. You can create one by using the [dotnet new console](../../core/tools/dotnet-new-sdk-templates.md#console) command or from Visual Studio.
```

Comment:
Should you remove the version specificity here? The latest version is .NET 7, this says .NET 6, and later this year it will be .NET 8. Just thinking out loud here.

Reply:
Done. In a future PR, I think we can remove the additional zone pivot for older versions.